### DISCUSSION TITLE
能不能增加自动网页全屏的功能？

### DISCUSSION THREAD

**Discussion by kidzgy** [the1812/Bilibili-Evolved#1994] [功能建议 / Ideas]
能不能增加自动网页全屏的功能？
并且可以分开带右侧评论和不带右侧评论的

**Comment by Coulomb-G** [comment]
使用脚本前请认真阅读使用说明

**Comment by kidzgy** [comment]
https://live.bilibili.com/6 已经设置了默认网页全屏，但是无效，在一般的视频界面是有效的。

**Reply by Geeyun-JY3** [reply]
说清楚是直播嘛……直播和普通视频是分开的，要有的话开关也是在直播部分。

重复：#965
